src/dashboard/Push/PushDetails.react.js

```diff
@@ -39,7 +39,13 @@ const EXP_STATS_URL = 'http://docs.parseplatform.org/ios/guide/#push-experiments
 let getMessage = (payload) => {
   if(payload) {
     let payloadJSON = JSON.parse(payload);
-    return payloadJSON.alert ? payloadJSON.alert : payload;
+		if (payloadJSON.alert.body) {
```

Comment:
There is no need to check the type of `payloadJSON.alert` like in the other changed file? Also, in case there is a body, could you also add the title?